bitcoin/src/blockdata/script.rs

```diff
@@ -800,6 +800,22 @@ pub struct Instructions<'a> {
 }
 
 impl<'a> Instructions<'a> {
+    /// Iterate over instructions in `script_bytes`
+    pub fn new(script_bytes: &'a [u8]) -> Self {
+      Self {
+        data: script_bytes.iter(),
+        enforce_minimal: false,
+      }
+    }
+
+    /// Iterate over instructions in `script_bytes` enforcing minimal pushes
+    pub fn new_minimal(script_bytes: &'a [u8]) -> Self {
```

Comment:
If we decide that we want this despite #1155 then I'd prefer these called `from_raw_bytes` and `from_raw_bytes_minimal`